packages/shared/src/utils/login/login.ts

```diff
@@ -4,14 +4,27 @@ import { CookieStorage, isStorageSupported, LocalStore } from '../storage/storag
 import { getAppId, domain_app_ids } from '../config/config';
 import { getStaticUrl, urlForCurrentDomain } from '../url';
 
+const escapeHtml = (unsafe: string): string => {
+    return unsafe.replace(/[<>]/g, match => {
+        switch (match) {
+            case '<':
+                return '&lt;';
+            case '>':
+                return '&gt;';
+            default:
+                return match;
+        }
+    });
+};
```

Comment:
Only escapes the metacharacters `<` and `>` within the redirect URL. Checked this with my forked repository codeQL analysis, the `DOM text reinterpreted as HTML` warnings were resolved

Reply:
what about this one @adrienne-deriv https://github.com/binary-com/deriv-app/security/code-scanning/44

Reply:
Both errors originated from this file `login.ts`, thus when I sanitised all of the return values in the `loginUrl` function, the errors are resolved within my codeQL analysis:
<img width="695" alt="Screenshot 2023-04-20 at 2 06 30 PM" src="https://user-images.githubusercontent.com/103016120/233272806-3c9baacc-fcb6-4801-ba3a-68fca13089ad.png">